indra/newview/skins/default/xui/en/menu_viewer.xml

```diff
@@ -2852,21 +2852,69 @@ function="World.EnvPreset"
         <menu_item_call
          label="Open..."
          name="Open...">
+          <menu_item_call.on_enable
+             function="EnableGLTF"/>
           <menu_item_call.on_click
            function="Advanced.ClickGLTFOpen" />
         </menu_item_call>
         <menu_item_call
          label="Save As..."
          name="Save As...">
+          <menu_item_call.on_enable
+             function="EnableGLTF"/>
           <menu_item_call.on_click
            function="Advanced.ClickGLTFSaveAs" />
         </menu_item_call>
         <menu_item_call
          label="Decompose..."
          name="Decompose...">
+          <menu_item_call.on_enable
+             function="EnableGLTF"/>
           <menu_item_call.on_click
            function="Advanced.ClickGLTFDecompose" />
         </menu_item_call>
+        <menu_item_call
+         label="Upload..."
+         name="Upload...">
+          <menu_item_call.on_enable
+             function="EnableGLTF"/>
+          <menu_item_call.on_click
+           function="Advanced.ClickGLTFUpload" />
+        </menu_item_call>
+      </menu>
+      <menu
+         create_jump_keys="true"
+         label="Video"
+         name="Video"
+         tear_off="true">
+        <menu_item_call
+         label="1080x1920"
+         name="1080x1920">
+          <menu_item_call.on_click
+           function="Advanced.ClickResizeWindow"
+            parameter="1080x1920"/>
+        </menu_item_call>
+        <menu_item_call
+         label="1920x1080"
+         name="1920x1080">
+          <menu_item_call.on_click
+           function="Advanced.ClickResizeWindow"
+            parameter="1920x1080"/>
+        </menu_item_call>
+        <menu_item_call
+         label="1280x720"
+         name="1280x720">
+          <menu_item_call.on_click
+           function="Advanced.ClickResizeWindow"
+            parameter="1280x720"/>
+          </menu_item_call>
+        <menu_item_call
+          label="720x1280"
+         name="720x1280">
+          <menu_item_call.on_click
+           function="Advanced.ClickResizeWindow"
+            parameter="720x1280"/>
+          </menu_item_call>
```

Comment:
Since it's the Advanced menu, it's fair game for new additions, but I'm curious: why where these buttons added to the Advanced menu?

Reply:
I'm whittling away at adding support for uploading videos to social media.  Step one is adding buttons that resize the viewer to preferred resolutions for places like Instagram.